.github/workflows/chromatic.yml

```
@@ -0,0 +1,26 @@
+name: 'Chromatic'
+
+on: push
+
+env:
+  NODE_VERSION: 16
+jobs:
+  chromatic-deployment:
+    runs-on: ubuntu-latest
```

No need to create another worklow yaml just add the following in the previous change you had done
```suggestion
    runs-on: ubuntu-latest
    if: github.event_name == 'push'
```

---

Want to keep it separate, else it just becomes unmanageable yaml